### ISSUE TITLE
Self-hosting the module, can't specify options

### ISSUE THREAD

**Issue by Nakou** [DiceBear/avatars#81]
Hello, thank you for this cool lib.
I have some issues tho;

I've implemented @dicebear/avatars-avataaars-sprites into a simple express nodejs app, and it works, I can get a random avatar.
But when I try to put options relative to the style and the elements of the avatar, like "topChance: 10", the application refuse to launch properly. The "option" wherever its placed (for new Avatar(sprites, options) nor avatars.create(seed, options)) is waiting for the default options only (height, width, etc...). Somehow it doesn't extends the avataaars version of options.

Is there something I dont understand or it's just not possible?

Here a snipped of the code I've tried to use : 
```
import express = require('express');
import Avatars from '@dicebear/avatars';
import sprites from '@dicebear/avatars-avataaars-sprites';

/* some code */

myApp.use('/example', (req, res)=>{
    var seed = req.url;
    let options = {};
    let avatars = new Avatars(sprites, options);
    var avatarsOptions = {
        topChance: 10
    }
    let svg = avatars.create(seed, avatarsOptions);
    res.send(svg);
});
```

Console output : 

```
TSError: ⨯ Unable to compile TypeScript:
server.ts:27:40 - error TS2345: Argument of type '{ topChance: number; }' is not assignable to parameter of type 'Options & Options'.
  Type '{ topChance: number; }' is not assignable to type 'Options'.
    Types of property 'topChance' are incompatible.
      Type 'number' is not assignable to type '100'.

27     let avatars = new Avatars(sprites, options);
                                           ~~~~~~~
```

Is there something missing? (in the documentation, in my code)
thanks in advance.

**Comment by FlorianKoerner** [comment]
Hi @Nakou 

The types from option "topChance" were wrong. I corrected that with version 4.2.3.
So an update should help. Can you confirm that?

https://codesandbox.io/s/adoring-kepler-cfd5i?file=/src/index.ts

**Comment by Nakou** [comment]
Thank you for your quick response.
The update partially worked for my case, it works now for the number-type options, but it doesn't recognize the array of strings type of options, like "top".

Here's what I tried and what I got in the console :
```
    var avatarsOptions = {
        topChance: 75,
        top: ['longHair', 'shortHair', 'hat'],
        facialHairChance:35
    }
    let svg = avatars.create(seed, avatarsOptions);
```

```
server.ts:44:40 - error TS2345: Argument of type '{ top: string[]; facialHair: string[]; facialHairChance: number; clothes: string[]; }' is not assignable to parameter of type 'Options & Options'.
  Type '{ top: string[]; facialHair: string[]; facialHairChance: number; clothes: string[]; }' is not assignable to type 'Options'.
    Types of property 'top' are incompatible.
      Type 'string[]' is not assignable to type '"longHair" | "shortHair" | "hat" | "eyepatch" | "hijab" | "turban"'.
        Type 'string[]' is not assignable to type '"turban"'.

44         let svg = avatars.create(seed, avatarsOptions);
                                          ~~~~~~~~~~~~~~
```

It do the same kind of errors for the options facialHair and clothes.

I am pretty new in NodeJS/Typescript so it's maybe a misunderstanding on how to do this kind of calls in TS/JS from my part and if it's the case, I apologize. If it's the case, maybe adding an option example in the documentation would be a wonderful thing, helping newbies like me :3.

Thanks in advance.

**Comment by FlorianKoerner** [comment]
Your example looks correct. The types are still wrong in parts. My mistake. I'll correct it in about eight hours.

In the meantime, you can use this workaround:

```ts
    var avatarsOptions = {
        topChance: 75,
        top: ['longHair', 'shortHair', 'hat'],
        facialHairChance:35
    }
    let svg = avatars.create(seed, avatarsOptions as any);
```

**Comment by FlorianKoerner** [comment]
Hopefully completely fixed with v4.2.5 😆
